src/app/pages/swap/form/swap-form.tsx

```diff
@@ -0,0 +1,25 @@
+import { Form, Formik } from 'formik';
+
+import { HasChildren } from '@app/common/has-children';
+
+import { SwapFormLayout } from './swap-form.layout';
+import { useSwapForm } from './use-swap-form';
+
+export function SwapForm({ children }: HasChildren) {
+  const { initialValues, validationSchema } = useSwapForm();
+
+  return (
+    <SwapFormLayout>
+      <Formik
+        initialValues={initialValues}
+        // Called explicitly in Swap
+        onSubmit={() => {}}
```

Comment:
Why is it called explicitly in swap?

Reply:
Let me look why it is there, maybe I can move it.